docs/UserGuide.md

```diff
@@ -238,8 +257,8 @@ A success message will be displayed if the supplier is successfully marked as ac
 
 ### Find a supplier: `find -s`
 
-The `find -s` command is used to find a supplier in VendorVault. 
-This helps you find suppliers based on keyword search.
+The `find -s` command is used to find a supplier in Vendor Vault. 
+This helps you to search for a supplier based on the supplier name, company, or product. 
 
 Format: `find -s n/NAME com/COMPANY pro/PRODUCT`
```

Comment:
I think can include a section for parameters to explain each of the parameters in the format stated

Reply:
Yup !